subsys/net/l2/ethernet/ethernet.c

```diff
@@ -122,6 +122,14 @@ static inline void ethernet_update_length(struct net_if *iface,
 					  struct net_pkt *pkt)
 {
 	uint16_t len;
+#ifdef CONFIG_NET_VLAN
+	const uint16_t min_len =
+		NET_ETH_MINIMAL_FRAME_SIZE - net_pkt_vlan_tag(pkt) != NET_VLAN_TAG_UNSPEC
+			? sizeof(struct net_eth_vlan_hdr)
+			: sizeof(struct net_eth_hdr);
```

Comment:
Could you add `()` so that it is easier to read this part, so something like
```
		NET_ETH_MINIMAL_FRAME_SIZE -
		(net_pkt_vlan_tag(pkt) != NET_VLAN_TAG_UNSPEC
			? sizeof(struct net_eth_vlan_hdr)
			: sizeof(struct net_eth_hdr));
```